rmm/monitor/src/rmi/realm.rs

```diff
@@ -0,0 +1,159 @@
+use crate::error::{Error, ErrorKind};
+use crate::listen;
+use crate::mainloop::Mainloop;
+use crate::realm::vm;
+use crate::rmi;
+
+extern crate alloc;
+
+pub fn set_event_handler(mainloop: &mut Mainloop<rmi::Receiver>) {
+    listen!(mainloop, rmi::Code::VMCreate, |call| {
+        info!("received VMCreate");
+
+        let vmm = vm::instance()
+            .ok_or(Error::new(ErrorKind::Unsupported))
+            .unwrap();
+        let id = vmm.create().unwrap();
+        call.reply(rmi::RET_SUCCESS)?;
+        call.reply(id)?;
```

Comment:
I think we can write like this.

```suggestion
        let realm = realm::create()?;
        call.reply(rmi::RET_SUCCESS)?;
        call.reply(realm.vmid())?;
```

Reply:
it means, I can remove the `unwrap()` ? or ?
after I will update changes, please suggest it again.